LayoutTests/TestExpectations

```diff
@@ -2648,6 +2648,9 @@ webkit.org/b/214290 imported/w3c/web-platform-tests/css/css-text/text-encoding/s
 webkit.org/b/214290 imported/w3c/web-platform-tests/css/css-text/text-encoding/shaping-tatweel-002.html [ ImageOnlyFailure ]
 webkit.org/b/214290 imported/w3c/web-platform-tests/css/css-text/text-encoding/shaping-tatweel-003.html [ ImageOnlyFailure ]
 webkit.org/b/214290 imported/w3c/web-platform-tests/css/css-text/text-indent/text-indent-tab-positions-001.html [ ImageOnlyFailure ]
+webkit.org/b/240838 imported/w3c/web-platform-tests/css/css-text/text-indent/anonymous-flex-item-001.html [ ImageOnlyFailure ]
+webkit.org/b/240838 imported/w3c/web-platform-tests/css/css-text/text-indent/anonymous-grid-item-001.html [ ImageOnlyFailure ]
```

Comment:
This bug was duped to [bug 240837](https://bugs.webkit.org/show_bug.cgi?id=240837)

Can you change it to match?

Reply:
Yeah those are weird, Safari was acting up when I submitted the new bug form so I had to submit it twice and I think both got through.